resources/build/minimum-versions.inc.sh

```diff
@@ -45,6 +45,6 @@ KEYMAN_DEFAULT_VERSION_UBUNTU_CONTAINER=noble # Ubuntu 24.04 Noble
 # Data versions -- see resources/standards-data/readme.md
 KEYMAN_VERSION_CLDR=45                        # LDML Keyboards version
 KEYMAN_VERSION_ISO639_3=2024-05-22            # Date of last import
-KEYMAN_VERSION_LANGTAGS=2024-05-22            # Date of last import
-KEYMAN_VERSION_LANGTAGS_SUBTAG_REGISTRY=2024-05-16  # Date of last import
+KEYMAN_VERSION_LANGTAGS=2025-02-18            # Date of last import
+KEYMAN_VERSION_LANGUAGE_SUBTAG_REGISTRY=2024-05-16  # Date of last import
```

Comment:
This just had an incorrect name. Wasn't referenced anywhere so safe to fix.

Reply:
Do you want to also get the 2025-02-06 update to language-subtag-registry on a separate PR?

Reply:
Yes, I think separate PR